samples/create-and-edit-geometries/src/main/java/com/esri/arcgismaps/sample/createandeditgeometries/screens/ButtonMenu.kt

```diff
@@ -154,6 +163,42 @@ fun ButtonMenu(
                 )
             }
         }
+        Box(
```

Comment:
You can just do 
```kotlin
Box { ... }
```
here as it has `modifier = Modifier` as a default parameter.